### DISCUSSION TITLE
Can I change the wind speed measurement unit?

### DISCUSSION THREAD

**Discussion by Marko-98** [breezy-weather/breezy-weather#659] [Q&A] [answered]
I prefer km/h to m/s and I haven't found a way to change it like in other apps.

**Comment by papjul** [comment, marked as answer]
Tap the gear icon on top right, it's one of the options listed under Appearance ;)

**Reply by Marko-98** [reply]
Thank you! I just found out I could change the icons as well. Thanks again. :)
